internal/env/supported_configurations.gen.go

```diff
@@ -133,7 +133,6 @@ var SupportedConfigurations = map[string]struct{}{
 	"DD_TRACE_128_BIT_TRACEID_LOGGING_ENABLED":                        {},
 	"DD_TRACE_ABANDONED_SPAN_TIMEOUT":                                 {},
 	"DD_TRACE_AGENT_PORT":                                             {},
-	"DD_TRACE_V1_PAYLOAD_FORMAT_ENABLED":                              {},
```

Comment:
The generation script moves this key to be in lexicographical order.

This change was made by running `go run ./scripts/configinverter/main.go generate`